README.md

```diff
@@ -96,3 +96,7 @@ Run "docker compose up" from the comand line
 ## Azure Link
 https://to-do-app-kl.azurewebsites.net/
 
+https://test-terratodoappkl.azurewebsites.net
+
+
+
```

Comment:
You need to add at least a section about Terraform setup and how to run it locally

Reply:
(don't forget about the environment variable setup)